src/app/notebook/shell/DatabaseManager.ts

```diff
@@ -351,11 +357,11 @@ export class DatabaseManager {
     return this._database?.exportJSON();
   }
 
-  async exportNotebook(peer: string) {
+  async exportNotebook(peer: string, topic: string) {
     const data = await this._database?.exportJSON();
     let toExport: Array<BlockDocument> = [];
     data?.collections[1].docs.forEach((element) => {
-      if (element.createdBy === peer) {
+      if (element.createdBy === peer && element.topic === topic) {
```

Comment:
The topic is an identifier for a set of blocks intended to represent a notebook, so it should be
```
if (element.topic === topic)
```

Reply:
Thank you for the clarification!